[live containers] add scubbing notes (#12959)
DataDog/documentation: content/en/infrastructure/livecontainers/configuration.md

- `credentials`
- `stripetoken`

You can set additional sensitive words by providing a list of words to the environment variable `DD_ORCHESTRATOR_EXPLORER_CUSTOM_SENSITIVE_WORDS`. This adds to, and does not overwrite, the default words. 

> **Note**: The additional sensitive words needs to be in lower-case as the agent compares the text with the pattern in lower-case as well. This means `password` would scrub `MY_PASSWORD` to `MY_*******` while `PASSWORD` would not.

Reviewer: ```suggestion
**Note**: The additional sensitive words must be in lowercase, as the Agent compares the text with the pattern in lowercase. This means `password` scrubs `MY_PASSWORD` to `MY_*******`, while `PASSWORD` does not.
```
